src/Sylius/Bundle/ImportExportBundle/Behat/ImportExportContext.php

```diff
@@ -0,0 +1,663 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ImportExportBundle\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use EasyCSV\Writer;
+use Sylius\Component\ImportExport\Model\ExportJobInterface;
+use Sylius\Component\ImportExport\Model\ExportProfileInterface;
+use Sylius\Component\ImportExport\Model\ImportJobInterface;
+use Sylius\Component\ImportExport\Model\ImportProfileInterface;
+use Sylius\Component\ImportExport\Model\JobInterface;
+use Sylius\Component\ImportExport\Model\ProfileInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Process\Process;
+use Sylius\Bundle\ResourceBundle\Behat\DefaultContext;
+
+/**
+ * ImportExportContext for ImportExport scenarios.
+ *
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
```

Comment:
I think you should add yourself to authors, cause you wrote pretty enormous amount of code here ;)